src/odemis/gui/main.xrc

```diff
@@ -5483,19 +5451,362 @@
                         <bg>#555555</bg>
                       </object>
                       <object class="FoldPanelItem">
-                        <object class="StreamBar" name="pnl_overview_streams">
```

Comment:
Just for your convenience, the xrc file changes related to the acquisition panel start form this line till line 5809